packages/styling/package.json

```diff
@@ -47,7 +47,7 @@
     "@microsoft/load-themed-styles": "^1.2.2",
     "@uifabric/utilities": ">=4.1.2 <5.0.0",
     "tslib": "^1.6.0",
-    "rtl-css-js": "^1.1.1",
+    "rtl-css-js": "^1.1.0",
```

Comment:
ahm might be also a bad merge, I'm not sure if this PR is in a good state at this moment